periphery/src/BorrowerLens.sol

```diff
@@ -1,19 +1,22 @@
 // SPDX-License-Identifier: AGPL-3.0-only
 pragma solidity ^0.8.15;
 
-import "aloe-ii-core/libraries/Uniswap.sol";
+import {Uniswap} from "aloe-ii-core/libraries/Uniswap.sol";
 
-import "aloe-ii-core/Borrower.sol";
+import {Borrower, IUniswapV3Pool} from "aloe-ii-core/Borrower.sol";
 
 contract BorrowerLens {
     using Uniswap for Uniswap.Position;
 
     function getLiabilities(Borrower account) external view returns (uint256 amount0, uint256 amount1) {
-        amount0 = Lender(account.LENDER0()).borrowBalance(address(account));
-        amount1 = Lender(account.LENDER1()).borrowBalance(address(account));
+        amount0 = account.LENDER0().borrowBalance(address(account));
+        amount1 = account.LENDER1().borrowBalance(address(account));
```

Comment:
This is a good callout. In general, `borrowBalance` is the more "correct" one because (as you said) it includes interest _up to and including the current block._ The reason I use `borrowBalanceStored` in the solvency check is that it's slightly more gas-efficient, and if that one additional chunk of interest-accrual is what pushes a user over the edge, liquidators can call `accrueInterest()` themselves. After calling `accrueInterest`, `borrowBalance == borrowBalanceStored`.

Here, since it's a Lens, the primary use-case is frontends fetching data -- so we don't have to worry about gas, and I can use the always-accurate `borrowBalace`